package.json

```diff
@@ -43,7 +43,8 @@
     "pushover-notifications": "0.2.0",
     "sgvdata": "0.0.2",
     "socket.io": "^0.9.17",
-    "git-rev": "git://github.com/bewest/git-rev.git"
+    "git-rev": "git://github.com/bewest/git-rev.git",
+    "browserify-express": "^0.1.4"
```

Comment:
consider using `npm install <dependency> --save` or manually slotting this alphabetically — it will simplify future diffs.